dashboard/src/shared/specs/mountWrapperIntl.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Components using the react-intl module require access to the intl context.
+ * This is not available when mounting single components in Enzyme.
+ * These helper functions aim to address that and wrap a valid,
+ * English-locale intl context around them.
+ */
+
+import { mount, shallow } from "enzyme";
+import * as React from "react";
+import { IntlProvider } from "react-intl";
+import { Provider } from "react-redux";
+import { BrowserRouter as Router } from "react-router-dom";
+import { MockStore } from "redux-mock-store";
+import I18n from "../I18n";
+
+// Default to english 1i8nconfiguration
+const messages = I18n.getDefaultConfig().messages;
+const locale = I18n.getDefaultConfig().locale;
+
+export const mountIntl = (node: React.ReactElement) =>
+  mount(
+    <IntlProvider locale={locale} key={locale} messages={messages} defaultLocale={locale}>
+      {node}
+    </IntlProvider>,
+  );
+
+export const mountWrapperIntl = (store: MockStore, children: React.ReactElement) =>
+  mount(
+    <Provider store={store}>
+      <IntlProvider locale={locale} key={locale} messages={messages} defaultLocale={locale}>
+        <Router>{children}</Router>
+      </IntlProvider>
+    </Provider>,
+  );
```

Comment:
I would just use the one with the `<Provider` for simplicity. Also, can we just add the `IntlProvider` to the existing wrapper at `specs/mountWrapper`? (That way we don't need to decide which wrapper to use but I'm not sure if that would break everything. It shouldn't)

Reply:
The issue is that many tests use the usual `mount` and `swallow` methods and these ones also need to have the intl context.

Regarding adding IntlProvider to the existing wrapper, yep, it is possible and it doesn't break anything (apart from some snapshots)